CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ Currently the versioning policy of this project follows [Semantic Versioning v2.
 
 ### Fixed
 - Fixed false positive UPM_UNCALLED_PRIVATE_METHOD for method used in JUnit's MethodSource ([[#2379](https://github.com/spotbugs/spotbugs/issues/2379)])
+- Use java.nio to load filter files ([[#2684](https://github.com/spotbugs/spotbugs/pull/2684)])
```

Comment:
Are you sure, you wanted to refer the PR here and not the fixed issue?

Reply:
The issue in the IntelliJ plugin, since Jetbrains does not seem to be actively maintaining it I think it will take a while until the plugin gets the updated version

Reply:
In this exact instance, I get it. However, I couldn't find the general logic yet, when exactly is referenced the issue, and when the PR, when there are no links? The links seems the same, when looking at the preview of the file (not the raw version), which may be a bit confusing for the reader. Until this point, I only refered the fixed issues. Could you please clarify this to me? (It's a bit off topic here, sorry about it.)

Reply:
I don't know either to be honest, but as a user I would expect a link to something giving some context as to what was changed and why